sklearn/cluster/k_means_.py

```diff
@@ -56,7 +56,8 @@ def _k_init(X, n_clusters, x_squared_norms, random_state, n_local_trials=None):
         Squared Euclidean norm of each data point.
 
     random_state : RandomState
-        The generator used to initialize the centers.
+        The generator used to initialize the centers. Pass an int for
+        reproducing the random values.
```

Comment:
I don't like how the wording has turned out. We could say "to reproduce", but "for reproducing" is awkward.

"Use an integer to make the randomness deterministic"??

Reply:
Sure, I will update to the following:
 **Use an int to make the randomness deterministic**